src/utils/types.ts

```diff
@@ -0,0 +1 @@
+export type InstanceType = "eval" | "standard";
```

Comment:
Sorry I didn't notice this earlier, this file should be in the folder of the feature. So `src/Kafka/KafkaInstanceDrawer/types.ts`

Given how it's used in a single place tho, is just make this type local to the component using it, no need to make it live in a single file or export it.

Reply:
We will have one for the service registry then. The two things are semantically different things even if they have the same values.